Screenbox/Pages/MainPage.xaml.cs

```diff
@@ -36,6 +38,8 @@ public sealed partial class MainPage : Page, IContentFrame
 
         private readonly Dictionary<string, Type> _pages;
 
+        private ISettingsService Settings => Ioc.Default.GetRequiredService<ISettingsService>();
```

Comment:
Getting the service directly from the control breaks the MVVM pattern. Ideally, we want only the ViewModels to have access to the services. Then the control will get the value from the ViewModel.